### DISCUSSION TITLE
storybook-vike

### DISCUSSION THREAD

**Discussion by Kenzo-Wada** [vikejs/vike#1596] [Discussion]
### Description

Hi @brillout 

Thanks a lot for maintaining such a awesome Framework!

WDYT about creating a integration plugin for storybook named like `storybook-vike` so that we can use APIs of vike inside stories file.
Im imagining something like Vike version of `@storybook/nextjs` .

**Comment by brillout** [comment]
Hi Kenzo, I'm glad you're enjoying Vike!

Sounds good :eyes: Would it require changes from Vike core?

**Comment by Kenzo-Wada** [comment]
> Would it require changes from Vike core?

I don't think it is necessary, and if it is needed in the future, I believe that storybook-vike should address Vike's destructive changes, not modify Vike!

P.S. I didn't know where to ask for advice, so I asked here, sorry if I'm in the wrong place.

**Comment by brillout** [comment]
:+1: Sounds good, we can give it a try and see. (My only concern would be complex Vike core changes that would slow down other high priority issues. We can try and see how it goes.)

No worries, moving this to GitHub Discussions.

**Reply by Kenzo-Wada** [reply]
@brillout 
Since the V1 design is still incomplete, I'd consider that a trade-off (maybe one of the ideas would be to start once V1 is released stable).

If we can start right away, how should we start: by creating a repository in Vike's organizing and grabbing npm?

**Reply by brillout** [reply]
> grabbing npm

:+1: Done. What's your npm handle? I'll make you co-owner.

> by creating a repository in Vike's organizing

How about we create the repository under your name at first?

**Reply by Kenzo-Wada** [reply]
Thank You! this is mine: https://www.npmjs.com/~kenzo-wada

> How about we create the repository under your name at first?

No problems on my side :)

**Reply by brillout** [reply]
Ok :)

> I'll make you co-owner.

Done.
